common/src/main/java/com/microsoft/identity/common/internal/platform/RawKeyAccessor.java

```diff
@@ -193,7 +213,7 @@ public boolean verify(@NonNull final byte[] text,
     }
 
     @Override
-    @Nullable
+    @CheckForNull
```

Comment:
Why change this? How is this different from Nullable and what is the benefit?

Reply:
Well, it's not part of androidx, for one.  Lombok doesn't have a corresponding @Nullable, and this is the one settled on by the JEP.

Reply:
Though it's been dormant for a while.  It depends on what we want to do with it.